certbot-apache/certbot_apache/constants.py

```diff
@@ -131,6 +131,16 @@
 MOD_SSL_CONF_DEST = "options-ssl-apache.conf"
 """Name of the mod_ssl config file as saved in `IConfig.config_dir`."""
 
+
+UPDATED_MOD_SSL_CONF_DIGEST = ".updated-options-ssl-apache-conf-digest.txt"
+"""Name of the hash of the updated or informed mod_ssl_conf as saved in `IConfig.config_dir`."""
+
+ALL_SSL_OPTIONS_HASHES = [
+    '2086bca02db48daf93468332543c60ac6acdb6f0b58c7bfdf578a5d47092f82a',
+    '4844d36c9a0f587172d9fa10f4f1c9518e3bcfa1947379f155e16a70a728c21a',
```

Comment:
And potentially for more convenience, I created the branch `actually-file-update-apache2` with what I think this PR should look like. I'd really like to know where you got the hash on this line as it's totally possible I missed something, but if you're happy with that branch, force push it here or open a new PR and I'll approve and merge it.